editor/src/components/canvas/ui-jsx-canvas.tsx

```diff
@@ -267,10 +283,68 @@ export const UiJsxCanvas = betterReactMemo(
       clearErrors()
     }
 
+    // TODO after merge requireFn can never be null
```

Comment:
I don't understand what this TODO is signifying?

Reply:
hah! I'll fix the todo. basically we can remove this IF from here, but that would de-indent the entire function making the PR garbage